### ISSUE TITLE
No luck trading

### ISSUE THREAD

**Issue by amexperts** [michaelgrosner/tribeca#137]
I am having a few issues with the different exchanges. 
On Bitfinex it connects fine however it is not showing my balance at all (I have it in margin wallet) and no quotes are being generated because 

`"cannot compute a quote since no position report exists!"`

So I tried Hitbtc and it generated quotes but even though I could see market orders etc it said it was disconnected to the exchange.

OKCoin kept throwing an error also (sorry I didnt save the error - but I believe it was to do with null hostname).

I have all my funds in Bitfinex now so would like to try and get this working.. any clues.. I am trying to trade with ETH/USD

Thanks

**Comment by amexperts** [comment]
![screen shot 2017-06-20 at 8 17 14 am](https://user-images.githubusercontent.com/28491705/27304067-02ef4ff8-5591-11e7-8a4b-3889ede9f9ca.png)
![screen shot 2017-06-20 at 6 47 20 am](https://user-images.githubusercontent.com/28491705/27304082-1203805e-5591-11e7-83fd-6ab54c8a928b.png)

**Comment by shanehull** [comment]
Having the same issue. I've only put 1XMR in there at this stage. Is there a minimum?

**Comment by MannieJay** [comment]
I was having similar issues with coinbase and I found this issue was related to an error I made while copying and changing the configuration file in etc. Make sure bot to remove anything and change all null values related to your exchange. I would recommend starting from scratch and reading every line in the config to make sure you don't miss one. Good luck! :)

**Comment by MannieJay** [comment]
Make sure *not to remove anything..

**Comment by shanehull** [comment]
Thanks @MannieJay. My config is looks correct and I'm able to connect to the exchange. I've tried configuring it again with no luck. The only thing I'm unsure about is the order destination.     '"BitfinexOrderDestination": "Bitfinex"' Should this be changed to something or is Bitfinex the correct value?

**Comment by shanehull** [comment]
This must have been an API issue. I started mine today with no changes and it began working.

**Comment by interwebcoding** [comment]
Yeah, seems to be API/Bitfinex Account related.  I have tested the same code / installation on  two separate API's/accounts.  One works, the other doesn't.

This is the one that is not working - logs returning : "cannot compute a quote since no position report exists!" is a new account that I havent used much yet.  

Anybody figure out what/why an API key wont work for getting positions?  @shed909 you said yours just started working, I see that it took you about 2 days, did you do anything on your bitfinex account?  Complete verification or anything like that?

**Comment by interwebcoding** [comment]
@amexperts did you ever get yours working?
